### PR TITLE
Add SSVM

### PR DESCRIPTION
**Pull request by alabulei1**
SSVM  is a WebAssembly runtime optimized for server-side applications. It replaces un-managed and unsafe code on the server-side, such as native code for high performance, AI, or big data operations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1633,8 +1634,8 @@ This repo contains a list of virtual machines and tools that execute the WebAsse
 
 
 
-## <a name="twvm"></a>[TWVM](https://github.com/Becavalier/TWVM) <sup>[top⇈](#contents)</sup>
-> A tiny and efficient WebAssembly virtual machine.
+## <a name="SSVM"></a>[TWVM](https://github.com/Becavalier/TWVM) <sup>[top⇈](#contents)</sup>
```
Comment (appcypher) [inline README.md]: Looks like you made a typo. Can you change this back to twvm?

Comment (appcypher) [inline README.md]: Looks like you made a typo. Can you change this back to twvm?

Comment (alabulei1) [reply README.md:1637]: Got it. Sorry for this.

Comment (alabulei1) [reply README.md]: I have fixed this. Thanks.

**Comment by appcypher** [comment]
LGTM

**Comment by alabulei1** [comment]
> LGTM

Thank you very much. The new version of SSVM will allow Node.js applications to call WebAssembly functions written in Rust or other high performance languages.

**Pull request merged**